plugin-tray/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ qt_add_dbus_adaptor(SOURCES org.kde.StatusNotifierItem.xml sniproxy.h SNIProxy)
 qt_add_dbus_interface(SOURCES org.kde.StatusNotifierWatcher.xml statusnotifierwatcher_interface)
```

Comment:
These changes to tray don't belong to the current PR. Please remove them.

Using of private headers isn't recommended, unless really inevitable. You could make a separate PR and explain in it what issue you intend to fix.

Reply:
Removed for now.